client/src/main/java/io/getstream/chat/android/client/socket/EventsParser.kt

```diff
@@ -63,39 +57,27 @@ internal class EventsParser(
     }
 
     private fun handleEvent(text: String) {
-        val eventMessage = parser.fromJsonOrError(text, TypedEvent::class.java)
-
-        if (eventMessage.isSuccess) {
-
+        val eventResult = parser.fromJsonOrError(text, ChatEvent::class.java)
```

Comment:
This seems like the correct way to parse it based on what we've discussed earlier, should fix some errors here 👍

Reply:
To understand this fix I have to see EventAdapter for GSON. It's a tricky thing :-)